### ISSUE TITLE
Audacity's team is censoring legitimate questions

### ISSUE THREAD

**Issue by AntiSol** [AntiSol/audacity#1]
This issue is for discussions on the original issue at https://github.com/audacity/audacity/issues/1346, which was nuked within 10 mins by the Gestapo.

**Comment by patemathic** [comment]
It seems as though they're deliberately trying to do as bad PR as possible.....it's almost funny at this point

**Comment by AntiSol** [comment]
Indeed. though it's weird: if you're *trying* to do bad PR, why [hire ars technica to do a(nother) positive write-up](https://arstechnica.com/gadgets/2021/07/audacitys-new-owner-is-in-another-fight-with-the-open-source-community/)?

Maybe they told Ars in the brief "make it look like a paid ad, not an unbiased article". They did kind of follow through on that.
